packages/headless/src/features/generated-answer/generated-answer-state.ts

```diff
@@ -0,0 +1,19 @@
+import {GeneratedAnswerCitation} from '../../api/generated-answer/generated-answer-event-payload';
+
+export interface GeneratedAnswerState {
+  isLoading: boolean;
+  answer?: string;
+  citations: GeneratedAnswerCitation[];
+  error?: {
+    message?: string;
+    code?: number;
+    isRetryable?: boolean;
+  };
+}
```

Comment:
Assuming later we could keep a state here of 

```
liked: boolean;
disliked: boolean;
```

Reply:
yeah i'll add that now